po/pt.po

```diff
@@ -0,0 +1,1701 @@
+# Portuguese (Portugal) translations for com.github.danrabbit.lookbook package.
+# Copyright (C) 2019 THE com.github.danrabbit.lookbook package.'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.danrabbit.lookbook package.
+# André Barata <andretiagob@protonmail.com>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.danrabbit.lookbook\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-10-25 12:09-0700\n"
+"PO-Revision-Date: 2020-08-27 19:13+0100\n"
+"Language-Team: none\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.1\n"
+"Last-Translator: ndré Barata <andretiagob@protonmail.com>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: pt\n"
+
+#: src/CategoryView.vala:38
+msgid "Actions"
+msgstr "Ações"
+
+#: src/CategoryView.vala:40
+msgid "Applications"
+msgstr "Aplicações"
+
+#: src/CategoryView.vala:42
+msgid "Categories"
+msgstr "Categorias"
+
+#: src/CategoryView.vala:44
+msgid "Devices"
+msgstr "Dispositivos"
+
+#: src/CategoryView.vala:46
+msgid "Emblems"
+msgstr "Emblemas"
+
+#: src/CategoryView.vala:48
+msgid "Emotes"
+msgstr "Emotes"
+
+#: src/CategoryView.vala:50
+msgid "Mimetypes"
+msgstr "Tipos Mime"
+
+#: src/CategoryView.vala:52
+msgid "Places"
+msgstr "Lugares"
+
+#: src/CategoryView.vala:54
+msgid "Status"
+msgstr "Estado"
+
+#: src/CategoryView.vala:56
+msgid "OTHER"
+msgstr "OUTRO"
+
+#: src/CategoryView.vala:74 src/IconView.vala:28
+msgid "Create a new address book"
+msgstr "Criar um novo livro de endereços"
+
+#: src/CategoryView.vala:79
+msgid "Align objects to center along the X axis"
+msgstr "Alinhar objetos ao centro ao longo do eixo X"
```

Comment:
I believe this may be more understandable:

```suggestion
msgstr "Alinhar objetos ao centro na direção horizontal"
```

(same for the other ones below)

Reply:
yup makes more sense :)